lib/expansion_tile_card.dart

```diff
@@ -95,64 +102,77 @@ class ExpansionTileCard extends StatefulWidget {
   /// Defaults to 2.0.
   final double elevation;
 
+  ///The elevetion when collapsed
```

Comment:
```suggestion
  /// The elevation when collapsed
```
Nit: Typo

Nit: Space before comments